lib/core/datatable/data/data-row.model.ts

```diff
@@ -19,6 +19,10 @@ export interface DataRow {
     isSelected: boolean;
     isDropTarget?: boolean;
     cssClass?: string;
+
     hasValue(key: string): boolean;
+
     getValue(key: string): any;
+
+    imageErrorResolver(event: Event): any;
```

Comment:
Should be nullable, otherwise will produce a breaking change for cases when people create rows like `<DataRow> { prop: value }`